src/main/scala/com/fulcrumgenomics/commons/util/DelimitedDataParser.scala

```diff
@@ -51,6 +51,12 @@ class Row private[util] (private val headerIndices: Map[String,Int], private val
   /** Fetches a value of the desired type by column name. */
   def apply[A](column: String)(implicit tag: ru.TypeTag[A]): A = apply(headerIndices(column))
 
+  /** Non-type-parameterized apply method that returns the column value as a String. */
+  def string(column: String): String = value(headerIndices(column))
+
+  /** Non-type-parameterized apply method that returns the column value as a String. */
+  def string(index: Int): String = value(index)
```

Comment:
I like `string` but it could also be named `raw`?

Reply:
I also wonder if you could check the typetag in apply to see if it is string, and then just short-circuit to return value, and if that would speed it up.

Reply:
I would really like to do the latter, but I couldn't find a way to check the type tag that was a) significantly faster than the existing implementation or b) anywhere close to as fast as a non-type-var'd method.

I'll stick with `string` if you don't mind since it still returns the trimmed string, so I think `raw` could be a touch misleading.